### DISCUSSION TITLE
Is this E8B7 LATIN SMALL LETTER LONG S WITH FLOURISH?

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#140] [Q&A] [answered]
I mean the first character on the enclosed scan (a fragment of Rubricella available at https://www.wbc.poznan.pl/dlibra/publication/493695):
![RubricellaUnglerX](https://user-images.githubusercontent.com/8024702/173759857-45a5da27-0593-446c-8a67-6d6d419505b1.png)
For me it looks more like the long s with et, but I don't see such a ligature neither in Unicode nor in MUFI.

**Comment by psb1558** [comment, marked as answer]
I think you're right: longs with et U+A76B. Unicode will never have it because it's a ligature, and I suppose MUFI never caught it, but Cappelli has it, 6th ed. p. 337:
<img width="468" alt="image" src="https://user-images.githubusercontent.com/16215767/173804285-20e8242d-22ac-4470-909f-109470b7c034.png">
Should Junicode have it?

**Reply by jsbien** [reply]
I would appreciate it very much.

**Comment by psb1558** [comment]
Reusing the et element of the q_et ligature. It will be in hlig.
<img width="387" alt="image" src="https://user-images.githubusercontent.com/16215767/173814592-cbef9123-3182-47ef-ae0e-a2128f7d4c76.png">

**Reply by jsbien** [reply]
Thanks!
